jupyterdrive/gdrive/drive-contents.js

```diff
@@ -113,7 +113,11 @@ define(function(require) {
             var metadata = values[0];
             var contents = values[1];
             var model = files_resource_to_contents_model(path, metadata);
-            model['content'] = notebook_model.notebook_from_file_contents(contents);
+            if(metadata.mimetyme == drive_utils.NOTEBOOK_MIMETYPE){
```

Comment:
on detecting the type (notebook vs text file):  We've already done this in ```file_resource_to_contents_model```.  ```model.type``` stores the file type.  But in this case the ```get``` function requires specifying the type anyway (is this optional?)

You might consider adding an optional parameter named ```type``` to ```file_resource_to_contents_model``` so that when the caller specifies the type, it will use the type specified by the user, and otherwise it will detect the file type.